daffodil-lib/src/main/scala/org/apache/daffodil/lib/xml/DaffodilXMLLoader.scala

```diff
@@ -251,26 +240,32 @@ class DFDLCatalogResolver private ()
         None
       }
       case (null, sysId) => {
-        // We did not get a catalog resolution of the nsURI, nor
-        // a straight file resolution of the systemId, so we
-        // use the systemId (which comes from the schemaLocation attribute)
-        // and the classpath.
-        val baseURI = if (baseURIString == null) None else Some(new URI(baseURIString))
-        val optURI = Misc.getResourceRelativeOption(sysId, baseURI)
+        // We did not get a catalog resolution of the nsURI. We now look for the systemID (which
+        // comes from the schemaLocation attribute) on classpath or as a file. Note that we
+        // ignore if boolean
```

Comment:
The sentence `Note that we ignore if boolean` is incomplete and I don't know what you mean by it.  Do you mean `Note that we ignore sysId's boolean value.`?